Backend/src/main/java/com/example/Backend/config/UserConfiguration.java

```diff
@@ -0,0 +1,7 @@
+package com.example.Backend.config;
+
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
Add the swagger config for learning. Refer - https://www.appsdeveloperblog.com/document-spring-rest-api-with-openapi-3swagger/